### PR TITLE
Add resources app

### PR DESCRIPTION
**Pull request by daniel-clayton**

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/apps.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        from cms.models import PageBaseSearchAdapter
+        from cms import sitemaps
+        Resource = self.get_model('Resource')
+
+        class ResourceSiteMap(sitemaps.PageBaseSitemap):
+            model = Resource
+
+            def items(self):
+                return super().items().filter(content__is_null=False)
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/apps.py]: You sure `is_null` as opposed to `isnull` will work? But `.exclude(content=None)` is simpler

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/models.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+    date = models.DateTimeField(
+        default=timezone.now,
+    )
+
+    class Meta:
+        ordering = ['-date', 'title']
+
+    def __str__(self):
+        return self.short_title or self.title
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/models.py]: `short_title` isn't present in the admin - add it to the admin (if for some reason there's a need for this) or remove it here

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/views.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    def get_unfiltered_queryset(self):
+        qs = super().get_queryset().filter(page__page=self.request.pages.current)
+
+        return qs
+
+    def get_queryset(self):
+        qs = self.get_unfiltered_queryset().select_related('type', 'image', 'page')
+
+        self.featured_resource = qs.first()
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/views.py]: This sets the featured resource to the first item in the queryset, but there's a `featured` field on `Resource` which is being ignored. Either remove it from the model or create a separate queryset with `.order_by('-featured')` to get candidates for the featured resource.

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/templates/resources/resource_list.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block main %}
+  <ul>
+    {% for object in object_list %}
+      <li>{{ object.render_item() }}</li>
+    {% else %}
+      <li>No resources</li>
+    {% endfor %}
+  </ul>
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/templates/resources/resource_list.html:10]: Stick a `render_pagination` in here as a reminder that this is a paginated view (it's easy to forget about `get_paginate_by` while building out the site)

**Review by lewiscollard — DISMISSED** [review]

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/apps.py (hunk 1 above)
Comment (joe-hastings) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/apps.py]: Changed in https://github.com/onespacemedia/project-template/pull/307/commits/1406ce5ab56ce5218e788776ef8bd72eb0312c44

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/models.py (hunk 2 above)
Comment (joe-hastings) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/models.py]: Changed in https://github.com/onespacemedia/project-template/pull/307/commits/1406ce5ab56ce5218e788776ef8bd72eb0312c44

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/views.py (hunk 3 above)
Comment (joe-hastings) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/views.py]: Changed in https://github.com/onespacemedia/project-template/pull/307/commits/1406ce5ab56ce5218e788776ef8bd72eb0312c44

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/templates/resources/resource_list.html (hunk 4 above)
Comment (joe-hastings) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/resources/templates/resources/resource_list.html:10]: Added in https://github.com/onespacemedia/project-template/pull/307/commits/1406ce5ab56ce5218e788776ef8bd72eb0312c44

**Review by joe-hastings — APPROVED** [review]
lgtm

**Pull request merged**